atomicapp/nulecule/base.py

```diff
@@ -100,6 +101,7 @@ def unpack(cls, image, dest, config=None, namespace=GLOBAL_CONF,
             docker_handler = DockerHandler(dryrun=dryrun)
             docker_handler.pull(image)
             docker_handler.extract(image, APP_ENT_PATH, dest, update)
+            Display().info("All dependencies installed successfully.", "cockpit")
```

Comment:
ahh I see. Yeah just makes it not seem uniform. Feel free to leave it like this or change it to make it clearer.